src/darwin/Framework/CHIP/MTRDevice_Concrete.mm

```diff
@@ -366,7 +384,7 @@ - (BOOL)isEqual:(id)object
 // better behavior.
 #define MTRDEVICE_SUBSCRIPTION_LATENCY_NEW_VALUE_WEIGHT (1.0 / 3.0)
 
-@interface MTRDevice ()
+@interface MTRDevice_Concrete ()
```

Comment:
We should duplicate the file for step 1, not rename.

That sets us up nicely for the next stage where we take all of the methods **in** MTRDevice and put warnings in them or logs, when they're called on the abstract version of the class

Reply:
Redid the first commit to duplicate the `.mm` file.